azext_edge/tests/edge/support/create_bundle_int/helpers.py

```diff
@@ -375,11 +375,12 @@ def get_all_kinds_from_manager(
     exclude_kinds = exclude_kinds or []
     result = set()
     for api in manager.resource_apis:
-        result.update(api.kinds)
+        if api.kinds:
+            result.update(api.kinds)
     return result - set(exclude_kinds)
 
 
-def get_file_map(
+def get_file_map(  # noqa: C901
```

Comment:
We should plan to resolve this at some point in the future.

Reply:
yeah this is a temporary flag